src/map/clif.cpp

```diff
@@ -1714,6 +1714,11 @@ int clif_spawn( struct block_list *bl, bool walking ){
 				clif_specialeffect(&md->bl,EF_GIANTBODY2,AREA);
 			else if(md->special_state.size==SZ_MEDIUM)
 				clif_specialeffect(&md->bl,EF_BABYBODY2,AREA);
+#if PACKETVER_MAIN_NUM >= 20200916 || PACKETVER_RE_NUM >= 20200724
+			struct block_list* master_bl = battle_get_master(&md->bl);
+			if (((mob_data*)bl )->special_state.ai == AI_ABR || ((mob_data*)bl )->special_state.ai == AI_BIONIC)
```

Comment:
Here you already have "md" you can use it directly for special_state access.
Then you dont need master_bl outside, but you can directly put it into function call.